src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -234,6 +235,7 @@ def load_arguments(self, _):
         c.argument('min_tls_version', help="The minimum version of TLS required for SSL requests, e.g., '1.0', '1.1', '1.2'")
         c.argument('http20_enabled', help="configures a web site to allow clients to connect over http2.0.", arg_type=get_three_state_flag(return_label=True))
         c.argument('app_command_line', options_list=['--startup-file'], help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
+        c.argument('ftps_state', help="Set the Ftps state value for an app. Default value is 'AllAllowed'. Other allwoed values are 'FtpsOnly' or 'Disabled'", arg_type=get_enum_type(FTPS_STATE_TYPES))
```

Comment:
Please double check the help. AFAIK, you don't need to call out the default one, because CLI's help system will automatically mention all the members, so you should remove the `Other allwoed values are 'FtpsOnly' or 'Disabled'` which has a typo anyway